Boot web permissions project

javierFernandezMendez opened the pull request.

javierFernandezMendez committed:
readme update

javierFernandezMendez committed:
permission not working yet

javierFernandezMendez committed:
db changes

javierFernandezMendez committed:
candidate table not showing

javierFernandezMendez committed:
database updated

pmkirsten committed:
Fixed permission path in DB

javierFernandezMendez committed:
using json file to save permissions

javierFernandezMendez committed:
routes permissions ready

javierFernandezMendez committed:
deail and new screens created

javierFernandezMendez committed:
permission example application finished with two users

javierFernandezMendez committed:
database fixed

javierFernandezMendez committed:
readme changed

javierFernandezMendez committed:
application title changed

daniel-grana commented on projectwiki-frontend/package.json:
```diff
@@ -36,6 +36,7 @@
     "ontimize-web-ngx": "8.11.1",
     "ontimize-web-ngx-theming": "8.5.1",
     "ontimize-web-ngx-report": "8.1.1",
+    "ontimize-web-ngx-filemanager": "8.5.6",
```
No se necesita para nada la dependencia del filemanager en esta demo no?

javierFernandezMendez committed:
dependencies fix

patricia-martinez-imatia committed:
Updated libraries and fixed error  Property 'deserialize' in type
'OntimizeMomentDateAdapter' is not assignable to the same property in base type 'MomentDateAdapter'.

pmkirsten reviewed (approved):
All ok

javierFernandezMendez committed:
extended ontimize service

javierFernandezMendez committed:
added new permissions to the demo user

javierFernandezMendez committed:
read json files method changed

javierFernandezMendez committed:
json reader method changed

javierFernandezMendez committed:
access denied route added and permission file reader method improved
